checkstyle.xml

```diff
@@ -141,4 +141,8 @@
   <module name="FileTabCharacter">
     <property name="eachLine" value="true"/>
   </module>
+  <module name="SuppressWarningsFilter" />
```

Comment:
we actually don't use checkstyle anymore

Reply:
Didn't know that. Maybe change it in the `CONTRIBUTING.md`?

Reply:
I missed that change as well. @LorenzoBettini I'd like to discuss it in #144.

Reply:
As I said, sorry for misunderstanding the use of checkstyle O:)